commands/info.js

```diff
@@ -204,12 +204,6 @@ module.exports.execute = async (client, message, args) => {
 					
 	}
 
-	const response = new Discord.MessageEmbed()
```

Comment:
Removing this didn't change the behavior of the command, right? Just confirming that this was tested

Reply:
As far as I can tell it doesn't, but there are other v12-related errors in this file I'm hoping to resolve later. Once those are fixed I'll be able to fully confirm if it does or does not change. Highly likely that it doesn't though.

Reply:
Sounds good!